packages/auth-ui/src/components/LoginForm.tsx

```diff
@@ -64,6 +67,7 @@ export function LoginForm({
             variant="outline" 
             className="w-full flex items-center justify-center gap-3"
             type="button"
+            onClick={openGoogleLogin}
           >
             <GoogleIcon className="w-5 h-5" />
             <span className="font-medium">Sign in with Google</span>
```

Comment:
We need to also make these buttons optional and fetch some info from the server from the social auth configuration. If Google tokens / keys are not configured, it should not show this button as an option, because otherwise most apps will have a broken google button unless they add the keys.